src/main/webapp/app/account/register/register.scss

```diff
@@ -0,0 +1,23 @@
+.registerTitle {
+  margin-bottom: 40px;
+}
+
+.strengthWrapper {
+  padding: 0;
+}
+
+.registerButton {
+  width: 100%;
+  margin-top: 40px;
+  height: 50px;
+  background: #5c88f7;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
+}
+
+.loginButton {
+  width: 100%;
+  margin-top: 20px;
+  height: 50px;
+  background: #f6f6f6;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
+}
```

Comment:
shouldn't this be on global since the styles of the buttons on login and register appear to be the same?